avm/src/vm/event.rs

```diff
@@ -20,13 +20,15 @@ pub enum BuiltInEvents {
   START,
   /// '__conn ' in ASCII or 5f5f 636f 6e6e 20(80)
   HTTPCONN,
+  NOP,
 }
 
 impl From<BuiltInEvents> for i64 {
   fn from(ev: BuiltInEvents) -> Self {
     match ev {
       BuiltInEvents::START => -9213673853036498142,
       BuiltInEvents::HTTPCONN => -9214243417005793441,
+      BuiltInEvents::NOP => i64::MAX,
```

Comment:
I do wonder if we should make it the negative max so it's negative like the rest of the events? That would also reduce the amount of edge case handling in the compiler since it should never emit a negative number for a custom event?